pkg/controller/nodes/array/execution_context.go

```diff
@@ -0,0 +1,48 @@
+package array
+
+import (
+	"strconv"
+
+	"github.com/flyteorg/flytepropeller/pkg/apis/flyteworkflow/v1alpha1"
+	"github.com/flyteorg/flytepropeller/pkg/controller/executors"
+)
+
+const (
+	FlyteK8sArrayIndexVarName string = "FLYTE_K8S_ARRAY_INDEX"
```

Comment:
would have loved for this and next to be a shorter string
I infact added the shorter strings in flytekit, but never ended up making changes in propeller. this is to reduce the size of the pod object itself